components/contribution-flow/PlatformTipContainer.tsx

```diff
@@ -171,55 +188,71 @@ export function PlatformTipContainer(props: PlatformTipContainerProps) {
+                );
+              },
+            }}
+          />
+        </P>
+        <div className="flex gap-5">
+          <div>
+            <FormattedMessage
+              defaultMessage="Your tip: <bold>{amount}</bold> ({percentage}%)"
+              values={{
+                bold: I18nBold,
+                amount: formatCurrency(props.value, props.currency as Currency, { locale: intl.locale }),
+                percentage: (percentage * 100).toFixed(2),
+              }}
+            />
+          </div>
+          {isCollapsed && props.selectedOption !== PlatformTipOption.NONE && (
+            <StyledLinkButton onClick={() => setIsCollapsed(false)}>
+              <FormattedMessage id="Edit" defaultMessage="Edit" />
+            </StyledLinkButton>
+          )}
+        </div>
+        <div className={clsx('mt-3', { hidden: isCollapsed || props.selectedOption === PlatformTipOption.NONE })}>
+          <PlatformTipInput
+            disabled={!props.amount}
+            amount={props.amount}
+            currency={props.currency}
+            selectedOption={props.selectedOption}
+            value={props.value}
+            onChange={props.onChange}
+          />
+        </div>
+        {(!isCollapsed || props.selectedOption === PlatformTipOption.NONE) && (
           <React.Fragment>
             <P mt="12px" fontWeight="400" fontSize="16px">
-              <FormattedMessage defaultMessage="Adding a platform tip helps us to maintain the platform and introduce new features." />
+              {message}
             </P>
-            <Flex mt="12px">
-              <Box flexGrow={1} fontWeight="700" fontSize="16px">
-                <FormattedMessage defaultMessage="Do you want to add a tip?" />
-              </Box>
-              <P fontStyle="italic">
-                <StyledLinkButton onClick={() => setIsWhyPlatformTipModalOpen(true)}>
-                  <FormattedMessage defaultMessage="Why?" />
-                </StyledLinkButton>
-              </P>
-            </Flex>
-            <Box mt="12px">
-              <PlatformTipInput
-                disabled={!props.amount}
-                amount={props.amount}
-                currency={props.currency}
-                selectedOption={props.selectedOption}
-                value={props.value}
-                onChange={props.onChange}
+            <div className="mt-3">
+              <StyledCheckbox
+                name="accept-payment-method-warning"
+                checked={percentage === 0}
+                onChange={({ checked }) =>
+                  checked
+                    ? props.onChange(PlatformTipOption.NONE, 0)
+                    : props.onChange(PlatformTipOption.FIFTEEN_PERCENT, 0.15 * props.amount)
+                }
+                label={
+                  <span className="text-base">
+                    <FormattedMessage defaultMessage="I don't want to contribute to Open Collective" />
+                  </span>
```

Comment:
According to https://www.figma.com/file/2gSaMuWjJIhzyFVC2OijNP/%5BOC.com%5D-Platform-features-%E2%9A%A1%EF%B8%8F?node-id=6099%3A10598&mode=dev, this text should be font-size: 14px, it's currently 16px